python/pyspark/sql/functions.py

```diff
@@ -1427,6 +1427,19 @@ def to_utc_timestamp(timestamp, tz):
     return Column(sc._jvm.functions.to_utc_timestamp(_to_java_column(timestamp), tz))
 
 
+@since(3.1)
+def timestamp_seconds(col):
```

Comment:
I vaguely remeber that pyspark can access scala SQL functions automatically. cc @HyukjinKwon

Reply:
There are two ways:
- Using `expr(...)`. I think we usually recommend this way when the SQL functions are not existent in `functions.scala`.
- We can simply add this into [this dictionary](https://github.com/apache/spark/blob/c7f2a9b323c5354c5dab1354c9a9bda19274dcdc/python/pyspark/sql/functions.py#L130-L136). This way is currently kind of discouraged. I was discussed in the mailing list before, and we should convert that dictionary into each function definition for better static analysis and IDE support.

Reply:
Since we're here, how common is the function? We shouldn't add if it's expected to be less common per:
https://github.com/apache/spark/blob/df2a1fe131476aac128d63df9b06ec4bca0c2c07/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L45

Reply:
Since we're here, how common is the function? We shouldn't add if it's expected to be less common per:
https://github.com/apache/spark/blob/df2a1fe131476aac128d63df9b06ec4bca0c2c07/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L45

Reply:
I see, so we need this func here.